### ISSUE TITLE
Update Data Buffer to handle various Inputs

### ISSUE THREAD

**Issue by jijoongmoon** [nnstreamer/nntrainer#19]
Currently Data Buffer works only with file I/O. In addition to this, it should generate data from feature extractor.

- [x] ~~input from raw files ( training set, validation set, test set )~~
- [x] ~~provides function pointer to get data with mini batch size~~
- [ ] take tflite model and directory location in which data is
- [x] nnstreamer interface ( get the data from nnstreamer interface )

**Comment by jijoongmoon** [comment]
input from raw file is done ( #50 )
provides function pointer to get data with mini batch size : done ( #54 )

**Comment by jijoongmoon** [comment]
re open when we need tflite input interface.
